pom.xml

```diff
@@ -113,4 +113,17 @@
       <scope>test</scope>
     </dependency>
   </dependencies>
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.sonarsource.sonar-packaging-maven-plugin</groupId>
```

Comment:
why did you add this section ? `pluginClass` is already declared in <properties>.